applications/crossbar/test/cb_storage_test.erl

```diff
@@ -3,18 +3,70 @@
 -include_lib("eunit/include/eunit.hrl").
 -include("cb_token_restrictions_test.hrl").
 
-%%%=============================================================================
-%%% Generators
-%%%=============================================================================
+-define(ACCOUNT_ID_UNDER_TEST, <<"account0000000000000000000000001">>).
+-define(ACCOUNT_DB_UNDER_TEST, <<"account%2Fac%2Fco%2Funt0000000000000000000000001">>).
+-define(ACCOUNT_DB_CURRENT_MONTH,
+        kazoo_modb:get_modb(kz_util:format_account_id(?ACCOUNT_ID_UNDER_TEST))).
 
-%%------------------------------------------------------------------------------
+%% =======================================================================================
+%% Fixtures / Generators
+%% =======================================================================================
+
+%%--------------------------------------------------------------------
 %% @doc
 %% @end
-%%------------------------------------------------------------------------------
+%%--------------------------------------------------------------------
 maybe_check_storage_settings_test_() ->
+    {setup
+    ,fun setup/0
+    ,fun cleanup/1
+    ,fun (_ReturnOfSetup) ->
+             maybe_check_storage_settings()
+     end
+    }.
+
+%% =======================================================================================
+%% Setup
+%% =======================================================================================
+
+%%--------------------------------------------------------------------
+%% @doc
+%% @end
+%%--------------------------------------------------------------------
+setup() ->
+    ?debugMsg(":: Setting up Kazoo FixtureDB"),
+    %% Required for s3 attachment handler
+    _ = application:ensure_all_started(erlcloud),
+    Pid = kz_fixturedb_util:start_me(),
+    meck:new(kz_fixturedb_db, [unstick, passthrough]),
+    meck:new(kz_fixturedb_doc, [unstick, passthrough]),
+    meck:expect(kz_fixturedb_db, db_exists, this_month_db_exists()),
+    meck:expect(kz_fixturedb_doc, open_doc, fun open_doc/4),
+    Pid.
+
+cleanup(Pid) ->
+    _ = application:stop(erlcloud),
+    kz_fixturedb_util:stop_me(Pid),
+    meck:unload(),
+    ?debugMsg(":: Stopped Kazoo FixtureDB").
+
+%% =======================================================================================
```

Comment:
It is the test file and I don't want nagging, but anyway I call this line comment separator. Generally after edocification, separator lines should be 80 characters in total.

This line (and 2 below) is used to separator code section. So usually I recommend using three `%` at beginning followed by 77 `=` (no space between `%` and `=`).

For comments in the module beginning, use `%%%` and 77 `-` for separator.
For function comments, use `%%` and 78 `-` for separator.

In all case do not add space between `%` and `-` or `=` character at line beginning.

This makes life so easy if later we want to use `grep`, `sed` and other bulk changing commands later.

Sorry I had to write explanation for commenting format before, but I didn't. I'll try to write it soon. Also `edocify` script is not a very good written code, I'll try to expand it to print more info of why it fails and where.